docs/kubernetes/kctlr-use-bigip-k8s.rst

```diff
@@ -178,7 +178,14 @@ Run :command:`kubectl describe` for any Node in the Cluster and make note of the
 Create a Kubernetes Node for the BIG-IP device
 ``````````````````````````````````````````````
 
-#. Create a Kubernetes Node resource.
+.. important::
+   :class: sidebar
+
+   You need to create a "dummy" Node in Kubernetes to make the Kubernetes API server aware of the BIG-IP device. The BIG-IP node's status will always display as "NotReady" because it is not actually a fully-participating Kubernetes Node. This status has no effect on the BIG-IP's ability to communicate in the overlay network.
+
+   Once you've added the BIG-IP "dummy" Node, you should be able to successfully send traffic through the BIG-IP system to and from endpoints within your Kubernetes Cluster. See :ref:`networking troubleshoot openshift` for tips to verify network connectivity.
```

Comment:
One last thing now that I see it officially: "Once you've added the BIG-IP 'dummy' Node and configured the BIG-IP tunnel, you should be able..."

Just don't want to give the impression that this is enough.

Reply:
Excellent point.